### ISSUE TITLE
My open orders is empty in exchange page after switching market

### ISSUE THREAD

**Issue by abitmore** [bitshares/bitshares-ui#3225]
**Describe the bug**
If a user has viewed settle orders E.G. in market BTS/bitCNY, then switching to a market that doesn't have settle orders E.G. in market BTS/GDEX.BTC, the "my open orders" tab will be empty since it still tries to show settle orders.

**To Reproduce**
1. Go to 'https://wallet.bitshares.org/#/account/exchange.btsbots'
2. Click on 'Exchange'
3. Click 'BTS' tab in 'my markets'
4. Click 'bitCNY' in the market list to view the bitCNY/BTS market
5. Scroll down to 'my open orders / settle orders'
6. Click 'Settle Orders'
7. Click 'GDEX.BTC' in the market list to view the 'GDEX.BTC/BTS' market
8. See an empty 'my open orders' tab

**Expected behavior**
When "settle orders" tab disappears, UI should switch to 'my open orders' tab correctly.

**Screenshots**
![image](https://user-images.githubusercontent.com/9946777/75616828-94cd0280-5b56-11ea-9f3f-d608d1c1e696.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04 LTS
 - Browser Firefox
 - Version 69

**Additional context**
I think I have submitted this issue before, but can't find it right now.

**Comment by abitmore** [comment]
Workaround for users when this happens: click the "my open orders" text then the orders will show.

**Comment by froooze** [comment]
@abitmore 
I could not reproduce this, maybe I have different default settings?

https://github.com/bitshares/bitshares-ui/pull/3194/commits/cbbeaf9fcd253d1b67611b9b44aa0e456692c78c

**Comment by abitmore** [comment]
> Does the problem still exist after this patch ? cbbeaf9

I have no testing environment so can't test, sorry.

**Comment by froooze** [comment]
> I have no testing environment so can't test, sorry.

You can also manually set to new default settings by cloning this:

![image](https://user-images.githubusercontent.com/40874087/75910335-00141e80-5e4e-11ea-911b-45c31eb2f658.png)

**Comment by abitmore** [comment]
By default, settlements and open orders are in the same group. Thus the bug.

**Comment by abitmore** [comment]
IMO changing the default values only hides the bug, since users can update the settings, then the bug report will come again but just with more steps to reproduce. Ideally UI should make sure it always work with whatever settings.

**Comment by froooze** [comment]
@abitmore 
I agree here, but the exchange layout gets a rewrite, changing the default settings is only the first step.

**Comment by froooze** [comment]
https://github.com/bitshares/bitshares-ui/issues/2906

**Comment by abitmore** [comment]
As I understood layout rewrite is not in the roadmap.

**Comment by froooze** [comment]
> As I understood layout rewrite is not in the roadmap.

at least on my roadmap

**Comment by abitmore** [comment]
IMHO the correct behavior should be: when the UI decides one panel should be hidden, it should activate another panel in the same group.
